README.md

```diff
@@ -54,27 +54,35 @@ Some systems provide pre-built packages:
   $ brew install s3fs
   ```
 
-Compilation
------------
+Compilation and installation from sources
+-----------------------------------------
 
-* On Linux, ensure you have all the dependencies:
+These are generic instructions to compile from the master branch, and should work on almost any GNU/Linux, Mac OS, BSD or similar.
```

Comment:
Stylize as macOS.

Reply:
Are you sure?

At the rest of the `README.md` file, it is `Mac OS`.

> # s3fs
> s3fs allows Linux and Mac OS X to mount an S3 bucket via FUSE. s3fs preserves the native object format for files, allowing use of other tools like s3cmd.

> # Installation
> - On Mac OS X, install via Homebrew:

I just now checked, and it seems `Mac OS` was the previous name, while currently it's `macOS`.

Maybe we can merge as it is, and then perform a `sed` (I see that at least `config.guess` still mentions `Mac OS X` as well).

Reply:
I believe Apple has renamed to macOS.  I can address this in a subsequent commits since it is widespread.

Reply:
It seems that Apple has changed "OS X(Mac OS)" to new name "macOS" from 10.12 Sierra.

Reply:
Fixed at #840